<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="cloudFileList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>781</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::CurrentChanged|QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed|QAbstractItemView::EditTrigger::SelectedClicked</set>
    </property>
   </widget>
   <widget class="QPushButton" name="UploadButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pathInput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectFile">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
    </property>
   </widget>
   <widget class="QTextEdit" name="currentServerpathInput">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>cloud path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backpathButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentRevert"/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJpCloud">
    <property name="title">
     <string>JpCloud</string>
    </property>
    <addaction name="actionconnect"/>
    <addaction name="action_disconnect"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>settings</string>
    </property>
    <addaction name="actionopensettings"/>
   </widget>
   <widget class="QMenu" name="RefreshDropdown">
    <property name="title">
     <string>Refresh</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
    </property>
    <addaction name="action_refresh"/>
   </widget>
   <addaction name="RefreshDropdown"/>
   <addaction name="menuJpCloud"/>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionconnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionopensettings">
   <property name="text">
    <string>opensettings</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>wd</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
